tests/_utils/start_tidb_cluster

```diff
@@ -18,8 +18,9 @@ cd $OUT_DIR && echo "start tidb cluster in $OUT_DIR"
 
 cat - > "$OUT_DIR/pd-config.toml" <<EOF
 [replication]
-# The number of replicas for each region.
-# max-replicas = 1
+# Set it to 1 to make sure we have enough replicas to run placement-rules.
+max-replicas = 1
+enable-placement-rules = true
```

Comment:
what will happen if `max-replicas` is set to 3 here

Reply:
Given the short time of running tests, it is possible to finish tests while replicas are still under 3. If so, the placement rule is not functioning properly. It would end with zero replicas on TiFlash nodes.